docs/ingestion/input-sources.md

```diff
@@ -1114,7 +1114,16 @@ This input source provides the following filters: `and`, `equals`, `interval`, a
 |type|Set this value to `not`.|yes|
 |filter|The iceberg filter on which logical NOT is applied|yes|
 
+`range` Filter:
 
+|Property|Description|Required|
+ |--------|-----------|---------|
+|type|Set this value to `range`.|yes|
+|filterColumn|The column name from the iceberg table schema based on which range filtering needs to happen.|yes|
+|lower|Lower bound value to match.|no. At least one of `lower` or `upper` must not be null.|
+|upper|Upper bound value to match. |no. At least one of `lower` or `upper` must not be null.|
+|lowerOpen|Boolean indicating if lower bound is open in the interval of values defined by the range (">" instead of ">="). |no|
```

Comment:
Would also be good to specify the defaults explicitly via a separate column or next to the "no" in the required column